routers/web/shared/user/header.go

```diff
@@ -144,6 +144,20 @@ func LoadHeaderCount(ctx *context.Context) error {
 	}
 	ctx.Data["RepoCount"] = repoCount
 
+	if !ctx.ContextUser.IsOrganization() {
+		starCount, err := repo_model.CountRepository(ctx, &repo_model.SearchRepoOptions{
+			Actor:              ctx.Doer,
+			Private:            ctx.IsSigned,
+			StarredByID:        ctx.ContextUser.ID,
+			Collaborate:        util.OptionalBoolFalse,
```

Comment:
```suggestion
			Collaborate:        optional.Some(false),
```

And import `code.gitea.io/gitea/modules/optional` above.